.ci/deps.sh

```diff
@@ -16,4 +15,4 @@ for dep_version in "${dep_versions[@]}" ; do
 
   pip3 install -q -r test-requirements.txt
   pip3 install -q -r requirements.txt
-done
\ No newline at end of file
+done
```

Comment:
emacs is doing the right thing, assuming you are the author of that file.

It is distracting to add an EOL at EOF in a patch which isnt about that.
So, check your diff before you PR, and undo this emacs fixup unless you are modifying the last line of the file.

Ideally we create a bear to require it (https://github.com/coala/coala-bears/issues/815), and then fix all the EOL problems at once.  Until then, ignore any missing EOLs.